libs/core-ui/src/lib/util/getDataBalanceChartOptions.ts

```diff
@@ -0,0 +1,287 @@
+        }`;
+      }
+    },
+    xAxis: {
+      categories: uniqueClasses,
+      title: { text: "Class B" }
+    },
+    yAxis: {
+      categories: uniqueClasses,
+      title: { text: "Class A" }
+    }
+  };
+}
+
+function getPointCategoryName(point: Point, dimension: "x" | "y"): string {
+  if (point.x === undefined || point.y === undefined) {
+    return "N/A";
+  }
+
+  const series = point.series,
+    isY = dimension === "y",
+    axis = series[isY ? "yAxis" : "xAxis"];
+  return axis.categories[isY ? point.y : point.x];
+}
+
+export function getDistributionBalanceChartOptions(
+  distributionBalanceMeasures: IDistributionBalanceMeasures,
+  datasetName?: string
+): IHighchartsConfig {
+  const approvedMeasureVarNames = new Set([
+    ...ApprovedDistributionMeasures.keys()
+  ]);
+
+  // Assume that every feature has the same measures computed
+  // and use the 1st feature to get a list of computed measures
+  const computedMeasures = Object.keys(
+    Object.values(distributionBalanceMeasures.measures)[0] // gets the measures dict for the 1st feature (i.e. 'race')
+  ); //  gets just the keys of the measures dict (i.e. ['inf_norm_dist', 'js_dist', 'kl_divergence'])
+
+  const measureOptions = computedMeasures
+    .filter((measureVarName) => approvedMeasureVarNames.has(measureVarName))
+    .map(
+      (measureVarName) =>
+        ApprovedDistributionMeasures.get(measureVarName)?.name ?? measureVarName
+    );
+
+  const features = distributionBalanceMeasures.features;
+  const data = features.map((featureName: string) => {
+    const measures = Object.entries(
+      getDistributionBalanceMeasures(
+        distributionBalanceMeasures.measures,
+        featureName
+      )
+    );
+    return {
+      data: measures
+        .filter(([measureVarNames]) =>
+          approvedMeasureVarNames.has(measureVarNames)
+        )
+        .map(([_, value]) => value),
+      name: featureName,
+      type: "column"
+    } as SeriesOptionsType;
+  });
+
+  const titleOptions = {
+    text: `Comparing ${features.join(", ")} with Uniform Distribution`
+  } as TitleOptions;
+  if (datasetName) {
+    titleOptions.text += ` in ${datasetName}`;
+  }
+
+  // TODO: Show the measure's description in a tooltip. Things tried but failed to work:
+  // - Appending to the default tooltip (could not access default tooltip text)
+  // - Overriding the default tooltip with a custom one (worked but could not render the colored circles that default tooltip has)
+  // - Adding a custom hover-over as a load() event (could not figure out which measure was being hovered over)
+  return {
+    chart: {
+      numberFormatter: (value: number) => value.toFixed(3),
+      type: "column"
+    },
+    series: data,
+    subtitle: {
+      text: `<a target="_blank" href="https://microsoft.github.io/SynapseML/docs/features/responsible_ai/Data%20Balance%20Analysis/#distribution-balance-measures">Reference Link for Distribution Balance Measures</a>`,
```

Comment:
It's weird for us to link to the SynapseML docs from this repository.  I wonder if this page can be replicated in the RAI toolbox documentation website.